### PR TITLE
feat(txn-fees): Update to bedrock

### PR DESCRIPTION
**Pull request by qbzzt**
- Update with the additional information from the bedrock wallet upgrade guide
- Remove "gas price too high" and "gas price too low" errors

Closing: [DEVRL-1001](https://linear.app/optimism/issue/DEVRL-1001)

### CODE REVIEW

File: src/docs/developers/build/transaction-fees.md [hunk 1]
```diff
@@ -21,15 +21,44 @@ This is exactly how fees work on Ethereum with the added bonus that gas prices o
...
 
 The amount of L2 gas used depends on the particular transaction that you're trying to send.
 Thanks to [EVM equivalence](https://medium.com/ethereum-optimism/introducing-evm-equivalence-5c2021deb306), transactions typically use approximately the same amount of gas on OP Mainnet as they do on Ethereum.
 Gas prices fluctuate with time and congestion, but you can always check the current estimated L2 gas price on the [public OP Mainnet dashboard](https://optimism.io/gas-tracker).
 
 
+
+### Base fee
+
+[The EIP 1559 parameters](./differences.md#eip-1559) have different values in OP Mainnet (and most other OP Stack chain) than those on L1 Ethereum.
```
Comment (sbvegan) [inline src/docs/developers/build/transaction-fees.md:36]: I don't see a reference to eip1559 in this link

File: src/docs/developers/build/transaction-fees.md [hunk 2]
```diff
@@ -21,15 +21,44 @@ This is exactly how fees work on Ethereum with the added bonus that gas prices o
...
+l2_execution_fee = transaction_gas_price * l2_gas_used
 ```
 
 The amount of L2 gas used depends on the particular transaction that you're trying to send.
 Thanks to [EVM equivalence](https://medium.com/ethereum-optimism/introducing-evm-equivalence-5c2021deb306), transactions typically use approximately the same amount of gas on OP Mainnet as they do on Ethereum.
 Gas prices fluctuate with time and congestion, but you can always check the current estimated L2 gas price on the [public OP Mainnet dashboard](https://optimism.io/gas-tracker).
 
 
+
+### Base fee
```
Comment (sbvegan) [inline src/docs/developers/build/transaction-fees.md:34]: This section might benefit from starting with a very short description of what the `base fee` is

File: src/docs/developers/build/transaction-fees.md [hunk 3]
```diff
@@ -21,15 +21,44 @@ This is exactly how fees work on Ethereum with the added bonus that gas prices o
...
+
+[The EIP 1559 parameters](./differences.md#eip-1559) have different values in OP Mainnet (and most other OP Stack chain) than those on L1 Ethereum.
+As a result, in every block the base fee can be between 98% and 110% of the previous value. 
+As blocks are produced every two seconds, the base fee can be between 54% and 1,745% of the value a minute earlier.
+If it takes the user fourteen seconds to approve the transaction in the wallet, the base fee can almost double in that time.
+
```
Comment (sbvegan) [inline src/docs/developers/build/transaction-fees.md:40]: I think this information is valuable, but I think it better served as a note or subsection under what the base fee is. Maybe something like, "A note about base fee volatility during periods of congestion"

**Review by sbvegan — COMMENTED** [review]
This is a great update, thanks Ori! Lmk if you have any questions about my comments

File: src/docs/developers/build/transaction-fees.md (hunk 1 above)
Comment (qbzzt) [reply src/docs/developers/build/transaction-fees.md:39]: There will be if we link merge https://github.com/ethereum-optimism/community-hub/pull/838 first.

File: src/docs/developers/build/transaction-fees.md (hunk 2 above)
Comment (qbzzt) [reply src/docs/developers/build/transaction-fees.md:34]: Added

**Review by sbvegan — APPROVED** [review]

**Review by OPMattie — APPROVED** [review]

**Pull request merged**